dali/test/python/test_operator_constant.py

```diff
@@ -162,3 +163,23 @@ def test_constant_fn():
 def test_scalar_constant_promotion():
     yield _test_scalar_constant_promotion, "cpu"
     yield _test_scalar_constant_promotion, "gpu"
+
```

Comment:
Maybe this should go to test_dali_variable_batch_size.py?

Reply:
It's just my opinion, but I don't like the whole idea of test_dali_variable_batch_size - it's a bag of very shallow tests. When we have variable batch size support across the board, we might just use variable batch size for new tests for other ops.

Reply:
So do we want to extend every operator's test to cover batch size variability more thoroughly?

Reply:
When it doesn't require jumping though too many hoops, then yes. Currently any test that uses readers is not exactly easily convertible to variable batch size.